src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Risk/HomeRiskCellModel.swift

```diff
@@ -304,8 +304,19 @@ class HomeRiskCellModel: CountdownTimerDelegate {
 			itemViewModels.append(recentRiskDaysModel)
 
 		} else if homeState.shouldShowDaysSinceInstallation {
+			var daysSinceInstallationTitle = ""
+
+			switch homeState.daysSinceInstallation {
```

Comment:
Please don't build logic that is already handled by the OS.
You might want to check out our `Localizable.stringsdict`